code/game/gamemodes/changeling/powers/mutations.dm

```diff
@@ -132,15 +132,6 @@
 	throw_range = 0
 	throw_speed = 0
 
-/obj/item/melee/arm_blade/New()
-	..()
-	if(ismob(loc))
-		loc.visible_message("<span class='warning'>A grotesque blade forms around [loc.name]\'s arm!</span>", "<span class='warning'>Our arm twists and mutates, transforming it into a deadly blade.</span>", "<span class='warning'>You hear organic matter ripping and tearing!</span>")
-
-/obj/item/melee/arm_blade/dropped(mob/user)
-	user.visible_message("<span class='warning'>With a sickening crunch, [user] reforms [user.p_their()] blade into an arm!</span>", "<span class='notice'>We assimilate the blade back into our body.</span>", "<span class='warning'>You hear organic matter ripping and tearing!</span>")
-	. = ..()
```

Comment:
Keeping this would be preferred.

Reply:
It does it double when you get the blade currently and it is already done like this when the blade is retracted by the ability.